rise-storage.html

```diff
@@ -1330,8 +1330,7 @@
         decodedUrl = url.substring(0, end !== -1 ? end : url.length);
       }
 
-      // Decode again to handle double slash encodings
```

Comment:
@donnapep what about this message? shouldn't we check the double slash encodings and do it twice if that is the case? cheers